index.html

```diff
@@ -171,12 +171,15 @@ <h2>Let's chat!</h2>
         <form action="https://formspree.io/f/mbjqaodr" method="POST" class="container">
           <ul>
             <li>
+              <label for="name">name</label>
               <input type="text" id="name" name="user_name" placeholder="Text Input" maxlength="30" required />
             </li>
             <li>
+              <label for="mail">mail</label>
               <input type="email" id="mail" name="user_email" placeholder="jeremy@gmail.com" required/>
             </li>
             <li>
+              <label for="msg">send a message</label>
               <textarea name="user_message" id="msg" placeholder="Enter text here" maxlength="500" required></textarea>
             </li>
             <li>
```

Comment:
- [ ] Kindly, refer to the Figma to correct this issue.
  * Put the text input field under the text correctly.
  * Change the placeholder Text Input to your names.
  * Arrange the submit button correctly, I suggest you to put it above the horizontal line.
![Selection_102](https://user-images.githubusercontent.com/55635977/126386527-0cd5095f-6509-4727-b163-afb630321535.png)